spec/helpers/firm_helper_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe FirmHelper, type: :helper do
+  let(:firm) { double(equity_release: 0) }
+
+  subject { helper.type_of_advice_list_item(firm, :equity_release) }
+
+  describe 'type_of_advice_list_item' do
+    context 'type value is not 100' do
```

Comment:
See this is what I mean really, it shouldn't be up to this helper to know the "special" meaning of these numbers.